packages/webview_flutter/webview_flutter_wkwebview/lib/src/legacy/web_kit_webview_widget.dart

```diff
@@ -655,7 +655,8 @@ class WebViewWidgetProxy {
       String keyPath,
       NSObject object,
       Map<NSKeyValueChangeKey, Object?> change,
-    )? observeValue,
+    )?
```

Comment:
The difference in formatting is most likely caused by the difference in Flutter versions. We develop our plugins on the `main` branch of Flutter, so you may need to run the formatter on the latest version. Or you can fix these manually.